02_activities/assignments/assignment.sh

```diff
@@ -21,21 +22,23 @@ unzip rawdata.zip
 # Complete assignment here
 
 # 1. Create a directory named data
-
+mkdir data
 # 2. Move the ./rawdata directory to ./data/raw
-
+mv rawdata data/raw
 # 3. List the contents of the ./data/raw directory
-
+ls -l data/raw
```

Comment:
So here it would be ./data/raw, which basically tells the computer "hey so you know the folder that I'm currently in? Yeah so go into the data folder, and then the raw folder." There is no issue with not using the "./", but it's a good practice to do so, so make it a habit of yours.